### ISSUE TITLE
How do you split the training and testing sets?

### ISSUE THREAD

**Issue by zhimi9999** [PKU-EPIC/DexGraspNet#4]
Hello, I am developing a grasping algorithm using your dataset. I would like to ask how are the training and test sets in your data set divided?

**Comment by mzhmxzh** [comment]
[train.txt](https://github.com/PKU-EPIC/DexGraspNet/files/11741848/train.txt) 4229 objects
[test.txt](https://github.com/PKU-EPIC/DexGraspNet/files/11741850/test.txt) 1126 objects
This is how we divided the dataset in our benchmark experiments. But you can also divide them differently.

**Comment by zhimi9999** [comment]
Did you merge all the datasets and randomly shuffle them? I have a strange result that the performance on the test set is actually better than that on the training set.

**Comment by mzhmxzh** [comment]
The train/test sets are indeed divided randomly, except for some categories that contain only a few instances, which are all sorted to the test set. 
So I cannot explain why your algorithm has a higher performance on the test set...
